### ISSUE TITLE
Theme editor is not UX friendly in grafana 7

### ISSUE THREAD

**Issue by nunux13** [yesoreyeram/yesoreyeram-boomtheme-panel#26]
Hello,

Thanks for your project.

My Debian distro upgraded to Grafana 7.1.5
The new layout of options on the right panel don't fit anymore and I can't edit the theme.

<img width="410" alt="Capture d’écran 2020-09-02 à 18 28 23" src="https://user-images.githubusercontent.com/28257369/92010444-1e4dcf00-ed4a-11ea-8577-6e185b4f3966.png">

The only way, i Founded was edit in the JSON view ... too bad ...
... Hope you can provide a fix ?

Best,

Pierre

**Comment by yesoreyeram** [comment]
Hi Pierre - I already seen this. With the new 7x, theme editor is not UX friendly. But there is a plan to upgrade the plugin to support 7.x with the new plugin architecture to align with new layout.

In the meantime, you can drag the editor to make it more spacious.

![image](https://user-images.githubusercontent.com/153843/92032812-9aeca780-ed62-11ea-8770-b73117b87ef6.png)

**Comment by nunux13** [comment]
Got'it ! You save my day.

I was trying to extend Left windows border, better to move the inner spliter border to the right to make more space ...

Many thanks,

Pierre

**Comment by yesoreyeram** [comment]
Glad it solved ur issue at least for now. Will plan for better UX next version.

Btw, I am interested to know how u r using the plugin. Any screenshot/showcase is much appreciated. That will help to improve the plugin and also inspiration for others

**Comment by nunux13** [comment]
I wrote a custom HTML status panel: lines of five 55px box colored RED or GREEN.
Values to display is stored in variables, and help to set the right red or green style.
That panel should display "as it is", even on mobile.
The regular grafana status panel component stretch the box to mobile screen wide, with 1 box per line.
I use your plugin in combination with my html code to inject the CSS red or green style.

Maybe I should take time to propose my own plugin composent.
But at least, I solved my need.
I can't share customer screen right now. Sorry for that.

Best,
